api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -50,9 +68,17 @@
                     allowableValues = ALLOWABLE_OUTPUT_FORMATS, defaultValue = "application/fhir" +
                     "+ndjson"
             )
-            @RequestParam(required = false, name = "_outputFormat") String outputFormat) throws IOException {
-        //String encoded = FHIRUtil.outcomeToJSON(FHIRUtil.getSuccessfulOutcome("OK"));
-        return new ResponseEntity<>(null, null,
+            @RequestParam(required = false, name = "_outputFormat") String outputFormat) {
+        //activeRequestCheck();
+
```

Comment:
Before we actually create a Job, let's apply validation to the request parameters. Let's make sure `_outputFormat` is one of the three accepted values and return 400.